src/Microsoft.OData.Core/JsonLight/ODataJsonLightContextUriParser.cs

```diff
@@ -299,7 +299,7 @@ private ODataPayloadKind ParseContextUriFragment(string fragment, Func<IEdmType,
                 {
                     detectedPayloadKind = ODataPayloadKind.ServiceDocument;
                 }
-                else if (fragment.Equals(ODataConstants.EntityReferenceCollectionSegmentName + "(" + ODataConstants.EntityReferenceSegmentName + ")"))
+                else if (fragment.Equals(ODataConstants.CollectionPrefix + "(" + ODataConstants.EntityReferenceSegmentName + ")"))

```

Comment:
>CollectionPrefix [](start = 56, length = 16)

this makes it seem like the name change on the constant introduces an inconsistency in naming convention. Why not change both constants or leave the old naming in place?